doc/GuideForCommitters.md

```diff
@@ -110,3 +110,75 @@ first add the `depends:doc` label to the OpenJ9 PR, and then ensure the contribu
 has opened an associated PR in the [openj9-docs](https://github.com/eclipse/openj9-docs) 
 repository. An OpenJ9 PR that requires documentation changes should not be merged 
 until the associated `openj9-docs` PR is also approved and ready to be merged.
+
+
+## Coordinated Merges With Eclipse OMR
+
+While changes requiring commits to occur in both the Eclipse OMR and
+OpenJ9 projects should ideally be staged to manage any dependencies,
+there are circumstances when this is not possible.  In such cases, a
+"coordinated merge" in both projects is required to prevent build
+breaks in the Eclipse OpenJ9 project.
+
+**The following can only be done by those who are committers in both
+the Eclipse OMR and Eclipse OpenJ9 projects.**
+
+Note: it is important to perform these steps with as little delay as
+possible between them.
+
+1.  Launch a pull request build from the OpenJ9 pull request that you
+    wish to merge that performs the same tests that the OMR Acceptance
+    build.  This ensures that the two changes put together would pass
+    an OMR Acceptance build if one were launched.
+    ```
+    Jenkins test sanity all jdk8,jdk11 depends eclipse/omr#{OMR PR number}
+    ```
+    Before proceeding with the coordinated merge, you must ensure that
+    the build is successful and that it was performed relatively
+    recently (generally within a day of a successful build).
```

Comment:
Ideally we would ensure the `openj9` and `master` branches of eclipse/openj9-omr are even. If they are not we risk promoting untested commits from OMR. Do you think it's worth noting that?

Reply:
Yes, agreed.  I had it in mind at the outset but forgot to write it down.  Fixed in forced push.